src/mapOfPeople.js

```diff
@@ -40,7 +40,15 @@
  * @return {Map}
  */
 function mapOfPeople(people) {
-  // write code here
+  const resultMap = new Map();
+  people.forEach(function(item, index) {
```

Comment:
rewrite as arrow functions better and use appropriate name for item of `people` array